### ISSUE TITLE
Feature Request: Easier UX

### ISSUE THREAD

**Issue by voltrare** [foobnix/LibreraReader#668]
making the design simpler because there is too many customization
or release a lightweight version that is focused for reading 

Or adopt Material UI

Sorry in advance

**Comment by pwd-github** [comment]
A more material design centric UI would be hugely beneficial IMO

**Comment by zLadin** [comment]
I agree. This app really deserves a better UI and UX considering its vast amount of useful features and customisability. Cureently, the UI is hard to navigate and hovering windows in general is not a great idea. Too much information on the ribbons whilst reading and some of them are unable to be disabled from settings

**Comment by viasux** [comment]
I would be willing to try and help with some wireframing or prototyping if this is planned at all.
